nixos-modules/valheim.nix

```diff
@@ -72,6 +72,49 @@ in {
       '';
     };
 
+    adminList = lib.mkOption {
+      type = with lib.types; listOf str;
+      default = [];
+      example = [
+        "76521191055385553"
+        "76551295065884553"
```

Comment:
Even if these IDs are not allocated today, if they are valid Steam IDs, they may be someday, and then we will have someone's Steam ID in the git history forever.  From [Valve's documentation](https://developer.valvesoftware.com/wiki/SteamID), it appears that it is possible to specify an "Invalid" Steam account type, in the account-type field.  Let's do that, and set the "Universe" field to "Individual / Unspecified".  Then the rest should not matter.

Reply:
I settled for the following IDs 72057602627862526 and 72057602627862527
```
72057602627862526:
|-------| |--| |------------------| |-----------------------------| |
000000001 0000 00000000000000000001 1111111111111111111111111111111 0
             ^
          invalid

72057602627862527:
|-------| |--| |------------------| |-----------------------------| |
000000001 0000 00000000000000000001 1111111111111111111111111111111 1
             ^
          invalid
```

The fourth part (from right to left) is the steam account type (4 bits). According to the [Valve source](https://developer.valvesoftware.com/wiki/SteamID) you provided, when it is 0 it is invalid and this value cannot be used (see table under "Types of Steam Accounts")